roles/openshift_control_plane/defaults/main.yml

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
Don't want this file.

Reply:
nope, i think that's the only new file that slipped in.